rust/src/detect/mod.rs

```diff
@@ -65,6 +110,9 @@ mod test {
         assert_eq!(TestEnum::BestValueEver.to_str(), "best_value_ever");
         assert_eq!(TestEnum::from_str("zero"), Some(TestEnum::Zero));
         assert_eq!(TestEnum::from_str("nope"), None);
-        assert_eq!(TestEnum::from_str("best_value_ever"), Some(TestEnum::BestValueEver));
+        assert_eq!(
+            TestEnum::from_str("best_value_ever"),
```

Comment:
Looks like `rustfmt` work that should be in its own commit

Reply:
If you feel strongly that it should be in a separate commit, I can make the update. There are a few other rustfmt changes in addition to this one.

Reply:
Makes reviewing easier